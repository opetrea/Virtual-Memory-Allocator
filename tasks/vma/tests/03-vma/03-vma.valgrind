==91833== Memcheck, a memory error detector
==91833== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==91833== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==91833== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==91833== Parent PID: 91832
==91833== 
==91833== 
==91833== HEAP SUMMARY:
==91833==     in use at exit: 0 bytes in 0 blocks
==91833==   total heap usage: 14 allocs, 14 frees, 8,504 bytes allocated
==91833== 
==91833== All heap blocks were freed -- no leaks are possible
==91833== 
==91833== For lists of detected and suppressed errors, rerun with: -s
==91833== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
