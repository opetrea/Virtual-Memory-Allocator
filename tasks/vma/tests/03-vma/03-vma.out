Total memory: 0xA bytes
Free memory: 0x9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x1 - 0x2
Miniblock 1:		0x1		-		0x2		| RW-
Block 1 end
Total memory: 0xA bytes
Free memory: 0x8 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 2

Block 1 begin
Zone: 0x1 - 0x3
Miniblock 1:		0x1		-		0x2		| RW-
Miniblock 2:		0x2		-		0x3		| RW-
Block 1 end
